README.md

```diff
@@ -74,7 +74,7 @@ see if some other plugin is using that mapping.
     * To use termite:<br/>
       `let g:gtfo#terminals = { 'unix' : 'termite -d' }`
     * To use rxvt-unicode:<br/>
```

Comment:
I don't think so. I don't use termite but its command name looks less unwieldy than that of rxvt-unicode, so that's probably what termite goes by on most Linux distributions.